src/connectors/voice-search/__tests__/connectVoiceSearch-test.js

```diff
@@ -232,80 +232,80 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/voice-searc
   });
 
   describe('getWidgetState', () => {
-    it('returns the same state if query is an empty string', () => {
-      const { widget, helper, refine } = getInitializedWidget();
-      const uiStateBefore = { foo: 'bar' };
-      refine('');
-      const uiStateAfter = widget.getWidgetState(uiStateBefore, {
-        searchParameters: helper.state,
-      });
-      expect(uiStateAfter).toBe(uiStateBefore);
-    });
+    test('returns the `uiState` empty', () => {
+      const { widget, helper } = getInitializedWidget();
```

Comment:
for other reviewers, I checked and `getInitializedWidget` doesn't call `getConfiguration`